pprint/src/main/scala/ammonite/pprint/PPrint.scala

```diff
@@ -30,11 +33,15 @@ object PPrint extends Internals.LowPriPPrint{
 case class PPrint[A](a: PPrinter[A], cfg: Config){
   def render(t: A): Iterator[String] = {
     if (t == null) Iterator("null")
-    else a.render(t, cfg)
```

Comment:
As a matter of style, I'd leave this on the same line because it's short enough (< 100 chars)